pkg/config/config.go

```diff
@@ -96,6 +97,16 @@ type Candlepin struct {
 	DevelOrg   bool   `mapstructure:"devel_org"` // For use only in dev envs
 }
 
+type SubsAsFeatures struct {
```

Comment:
i'm wondering if we might need to have fields like these set in the deployment.yaml for subs_as_features too, i *think* the secretKeyRefs would be the same:
```
  - name: CLIENTS_CANDLEPIN_SERVER
    value: ${CLIENTS_CANDLEPIN_SERVER}
  - name: CLIENTS_CANDLEPIN_CLIENT_CERT
    valueFrom:
      secretKeyRef:
        name: content-sources-candlepin
        key: cert
        optional: true
  - name: CLIENTS_CANDLEPIN_CLIENT_KEY
    valueFrom:
      secretKeyRef:
        name: content-sources-candlepin
        key: key
        optional: true
  - name: CLIENTS_CANDLEPIN_CA_CERT
    valueFrom:
      secretKeyRef:
        name: content-sources-candlepin
        key: ca
        optional: true
```

Reply:
yes! thanks

Reply:
oh we may need the parameter set too at the bottom of the deployment.yaml:

```
- name: CLIENTS_FEATURE_SERVICE_SERVER
  value: 'https://subscription.stage.api.redhat.com/svcrest/subscription/v5/features'
``` 

though the endpoint to query whether a user has access to certain features is different: https://subscription.stage.api.redhat.com/svcrest/subscription/v5/featureStatus. what do you think about setting this value to https://subscription.stage.api.redhat.com/svcrest/subscription/v5 so we can use it for your API and to query the other endpoint?